<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1243</width>
    <height>601</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="routeLabel">
          <property name="text">
           <string>Fahrstraßen:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="routeListWidget">
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="routeButtonLayout">
        <item>
         <widget class="QPushButton" name="clearRoute">
          <property name="text">
           <string>Auflösen</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="clearAllRoutes">
          <property name="text">
           <string>Alles auflösen</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="sectionLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Gleisabschnitte:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="sectionListWidget">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>1</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="sectionButtonLayout">
        <item>
         <widget class="QPushButton" name="clearSection">
          <property name="text">
           <string>Auflösen</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="clearAllSections">
          <property name="text">
           <string>Alles auflösen</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="rightLayout">
      <item>
       <widget class="QTabWidget" name="regionTabWidget">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="actionLayout">
        <item>
         <widget class="QPushButton" name="tourLeftPushButton">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Fahrstraße links</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="shuntLeftPushButton">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Rangieren links</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="extendPushButton">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Erweitern</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="shuntRightPushButton">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Rangieren rechts</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="tourRightPushButton">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Fahrstraße rechts</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1243</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDaten">
    <property name="title">
     <string>Daten</string>
    </property>
    <addaction name="actionBeenden"/>
   </widget>
   <widget class="QMenu" name="menuEditieren">
    <property name="title">
     <string>Editieren</string>
    </property>
    <addaction name="actionRechts"/>
    <addaction name="actionLinks"/>
    <addaction name="actionHoch"/>
    <addaction name="actionRunter"/>
    <addaction name="separator"/>
    <addaction name="actionTeilschritt"/>
    <addaction name="actionErweitern"/>
    <addaction name="actionNeigung"/>
   </widget>
   <addaction name="menuDaten"/>
   <addaction name="menuEditieren"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionBeenden">
   <property name="text">
    <string>Beenden</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionRechts">
   <property name="text">
    <string>Rechts</string>
   </property>
   <property name="shortcut">
    <string>Right</string>
   </property>
  </action>
  <action name="actionLinks">
   <property name="text">
    <string>Links</string>
   </property>
   <property name="shortcut">
    <string>Left</string>
   </property>
  </action>
  <action name="actionHoch">
   <property name="text">
    <string>Hoch</string>
   </property>
   <property name="shortcut">
    <string>Up</string>
   </property>
  </action>
  <action name="actionRunter">
   <property name="text">
    <string>Runter</string>
   </property>
   <property name="shortcut">
    <string>Down</string>
   </property>
  </action>
  <action name="actionNeigung">
   <property name="text">
    <string>Neigung</string>
   </property>
   <property name="shortcut">
    <string>I</string>
   </property>
  </action>
  <action name="actionTeilschritt">
   <property name="text">
    <string>Teilschritt</string>
   </property>
   <property name="shortcut">
    <string>O</string>
   </property>
  </action>
  <action name="actionErweitern">
   <property name="text">
    <string>Erweitern</string>
   </property>
   <property name="shortcut">
    <string>X</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
